### DISCUSSION TITLE
UI

### DISCUSSION THREAD

**Discussion by JElgar** [White-Rabbit-Entertainment/game#110] [Ideas]
How do we feel about something like this for the UI?

![Screenshot from 2021-03-12 09-31-53](https://user-images.githubusercontent.com/10330042/110920898-d2419f80-8315-11eb-8ae3-985f0c284adc.png)

**Comment by JElgar** [comment]
The yellow boxes on the left and right are an alternative to the "inventory" in the middle. That would allow you to have one held item and if you are a traitor also have on poison.

**Comment by erwingame** [comment]
Good
